2fa-api/handlers/verify.js

```diff
@@ -0,0 +1,28 @@
+const twilio = require("../twilio");
+
+const twilioClient = twilio.twilioClient;
+
+const verify = function (req, res) {
+  try {
+    const { OTP } = req.query;
+    const { phone } = req.query;
+
+    twilioClient.verify
+      .services(process.env.TWILIO_SERVICE_SID)
+      .verificationChecks.create({ to: `+1${phone}`, code: OTP })
+      .then((response) => {
+        req.session.loggedIn = response.valid;
+
+        return res
+          .status(200)
+          .json({ isValid: req.session.loggedIn, sessionId: req.sessionID });
+      })
+      .catch((error) => {
+        throw new Error(error);
```

Comment:
can you see why returning the raw error message is a bad idea? what could you do instead?

Reply:
It could potentially return sensitive customer information. A better Idea is to manually return a specific error message.